src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -59,7 +54,7 @@ const propTypes = {
 const defaultProps = {
     forwardedRef: () => {},
     report: {},
-    blockedFromConcierge: {},
+    emojiPickerDisabled: false,
```

Comment:
`shouldDisableEmojiPicker` or maybe `shouldShowEmojiPicker` is more in line with how we have stated in the style guide to set up boolean props

https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#boolean-variables-and-props

Reply:
I went with `shouldDisableEmojiPicker` because trying to flip the logic to be positive (ie. should show) made it pretty confusing to reason about with this:

```
(ReportUtils.chatIncludesConcierge(this.props.report) && User.isBlockedFromConcierge(this.props.blockedFromConcierge))
                            || ReportUtils.isArchivedRoom(this.props.report)
```